platform/admin/core/admin-core-migration-commands/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -28,7 +28,9 @@
             <action class="org.codice.ddf.migration.commands.ExportCommand">
                 <argument ref="configurationMigrationService"/>
                 <argument ref="security"/>
-                <argument ref="defaultExportDirectoryPath"/>
+                <argument value="${ddf.home}"/>
+                <argument value="etc"/>
+                <argument value="exported"/>
```

Comment:
@coyotesqrl These arguments break the container (even with using var-args on the Java side)